pom.xml

```diff
@@ -2031,7 +2031,7 @@
         <carbon.consent.mgt.version>2.2.12</carbon.consent.mgt.version>
 
         <!--Identity Governance Version-->
-        <identity.governance.version>1.4.39</identity.governance.version>
+        <identity.governance.version>1.4.41-SNAPSHOT</identity.governance.version>
```

Comment:
need to update the identity-governance version after merging https://github.com/wso2-extensions/identity-governance/pull/428